packages/devkit/src/executors/read-target-options.ts

```diff
@@ -15,7 +15,7 @@ export function readTargetOptions<T = any>(
   { project, target, configuration }: Target,
   context: ExecutorContext
 ): T {
-  const projectConfiguration = context.workspace.projects[project];
+  const projectConfiguration = context.projectsConfigurations.projects[project];
```

Comment:
This property doesn't exist in v15.0.0, so this change would break devkit compat with v15.0.0